hmac_example_test.go

```diff
@@ -25,18 +24,17 @@ func init() {
 
 // Example creating, signing, and encoding a JWT token using the HMAC signing method
 func ExampleNew_hmac() {
-	// Create a new token object, specifying signing method and the claims
-	// you would like it to contain.
-	token := jwt.NewWithClaims(jwt.SigningMethodHS256, jwt.MapClaims{
-		"foo": "bar",
-		"nbf": time.Date(2015, 10, 10, 12, 0, 0, 0, time.UTC).Unix(),
-	})
+	// Create a new token object, specifying signing method
```

Comment:
Actually I think it would make more sense to just rename the example function instead. In this example, `NewWithClaims` should be the preferred way to create the token. In the majority of the cases you want to add custom claims to your token, so this should be the "default" example.